src/services/ldap-config/hooks/index.js

```diff
@@ -0,0 +1,31 @@
+const { authenticate } = require('@feathersjs/authentication');
+const { iff, isProvider, disallow } = require('feathers-hooks-common');
+
+const { populateCurrentSchool } = require('../../../hooks');
+const fillDefaultValues = require('./fillDefaultValues');
+const restrictToSchoolSystems = require('../../ldap/hooks/restrictToSchoolSystems');
+
+module.exports = {
+	before: {
+		all: [authenticate('jwt')],
+		get: [],
+		create: [
+			// only allow external calls (the service needs a user)
+			iff(!isProvider('external'), disallow()),
+			populateCurrentSchool,
```

Comment:
Should it be checked that only admins with corresponding permissions may execute this function?

Reply:
this is checked by the systems service